config/oracle.ts

```diff
@@ -158,6 +158,13 @@ export default async function (hre: HardhatRuntimeEnvironment): Promise<OracleCo
             heartbeatDuration: (24 + 1) * 60 * 60,
           },
         },
+        EIGEN: {
+          priceFeed: {
+            address: "0xD3631AC9D81eD560D61957a55E9c992cdE497eb6",
```

Comment:
may not be needed, as i am unsure if there will be an on-chain feed for EIGEN

would be good to set the placeholder address to "FIXME", to prevent accidentally sending an incorrect configuration